x-pack/plugins/observability_solution/apm/public/components/app/entities/charts/log_error_rate_chart.tsx

```diff
@@ -135,8 +135,7 @@ export function LogErrorRateChart({ height }: { height: number }) {
         showAnnotations={false}
         fetchStatus={status}
         timeseries={timeseries}
-        yLabelFormat={yLabelAsPercent}
-        yDomain={{ min: 0, max: 1 }}
+        yLabelFormat={asInteger}
```

Comment:
+1

Reply:
Oh, I see, I hadn’t noticed the PR during development!
But shouldn’t it always be an integer? Since it’s a documents count, I didn’t consider the possibility of it being a decimal. Is it possible for it to be a decimal?

Reply:
Thanks for sharing the official documentation to back up the decision!

Reply:
I'll update the Log Rate chart to use `asInteger` as well, since it's also using `asDecimalOrInteger`.